src/components/todoList/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import TodoItem from "../todoItem";
+
+import { useSelector } from "react-redux";
+import { RootState } from "../../store/store";
+
+import "./style.css";
+
+const TodoList: React.FC = () => {
+  const todos = useSelector((state: RootState) => state.todoList.todos);
+  return (
+    <ol className="todoList">
+      {todos.length > 0 ? (
+        todos.map((todo, index) => <TodoItem key={index} index={index} />)
```

Comment:
You are passing index twice, isn't it un-necessary? As booth key and index will have the same value.